src/rno-g-acq.c

```diff
@@ -578,20 +578,18 @@ int flower_initial_setup()
 }
 
 
+static int record_timimg()
```

Comment:
if you're not going to return anything, maybe make this void? 
Or did you want to return the return value of system?

Reply:
good catch